Scheduler/observing_blocks_auxtel/block-295-latiss_daily_calibrations.json

```diff
@@ -218,23 +170,6 @@
                     12.8
                 ]
             }
-        },
```

Comment:
can you please also remove the script just above this one, that runs the daily_sflat?